CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - Don't redact clipped views (#4325)
 - Session replay for crash not created because of a race condition (#4314)
 - Double-quoted include, expected angle-bracketed instead (#4298)
+- Exclude the directory we use in NSApplicationSupportDirectory from device backups (#4335)
```

Comment:
`m`: I think this needs to be updated now

Reply:
Thanks!